config/pom.xml

```diff
@@ -60,6 +60,10 @@
             <groupId>mysql</groupId>
             <artifactId>mysql-connector-java</artifactId>
         </dependency>
+        <dependency>
```

Comment:
Please do not depend other driver. Please add the driver to classpath by yourself